doc/nrf/gs_programming.rst

```diff
@@ -135,7 +134,7 @@ Complete the following steps to build |NCS| projects on the command line after c
 
       .. note::
 
-	     To build from a directory other than the sample or application directory, run the west build command with an additional parameter *directory_name*,  specifying the sample or application directory.
+	     To build from a directory other than the sample or application directory, run the west build command with an additional parameter ``directory_name``,  specifying the sample or application directory.
```

Comment:
```suggestion
	     To build from a directory other than the sample or application directory, run the west build command with an additional parameter *directory_name*,  specifying the sample or application directory.
```
This is a variable, not an actual parameter. So you're supposed to run west build -b THEBOARD THEDIRECTORY, not west build -b THEBOARD directory_name